models/foodprint_qrcode_product_attributes.js

```diff
@@ -25,6 +25,10 @@ module.exports = function (sequelize, DataTypes) {
         type: DataTypes.STRING(255),
         allowNull: true,
       },
+      qrcode_hashid: {
```

Comment:
@DanielmAaron Same here, this should be a string not a BLOB. Also will be needed in the migration or else create a separate migration for this.